ruby/iot_sub.rb

```diff
@@ -0,0 +1,6 @@
+mqtt = ably.channels.get('device:rocket:1')
+mqtt.subscribe('message') do |payload|
```

Comment:
This snippet should demonstrate using an MQTT client to connect to Ably like the [equivalent JavaScript snippet](https://github.com/ably/homepage-snippets/blob/main/javascript/iot-sub.js), for example using the [mqtt gem](https://github.com/njh/ruby-mqtt):

```ruby
mqttClient.subscribe('device:rocket:1')
mqttClient.get do |topic, message|
  if topic == 'action'
    rocket.process_instruction(JSON.parse(message))
  end
end
```